repo2docker/buildpacks/conda/activate-conda.sh

```diff
@@ -4,8 +4,8 @@ test -f $CONDA_PROFILE && . $CONDA_PROFILE
 if [[ "${KERNEL_PYTHON_PREFIX}" != "${NB_PYTHON_PREFIX}" ]]; then
     # if the kernel is a separate env, stack them
     # so both are on PATH
-    conda activate ${KERNEL_PYTHON_PREFIX}
-    conda activate --stack ${NB_PYTHON_PREFIX}
+    mamba activate ${KERNEL_PYTHON_PREFIX}
+    mamba activate --stack ${NB_PYTHON_PREFIX}
 else
-    conda activate ${NB_PYTHON_PREFIX}
+    mamba activate ${NB_PYTHON_PREFIX}
```

Comment:
Shouldn't we only be using mamba for the installation commands, not list/clean/etc.?